src/test/java/eu/yals/test/ui/appinfo/PublicInfoTestIT.java

```diff
@@ -0,0 +1,28 @@
+package eu.yals.test.ui.appinfo;
+
+import eu.yals.test.ui.AppInfoPageTest;
+import org.junit.Test;
+import org.springframework.stereotype.Component;
+
+/**
+ * Checking elements of public info area with information about version
+ *
+ * @since 2.7
+ */
+@Component
+public class PublicInfoTestIT extends AppInfoPageTest {
+
+    @Test
+    public void publicAreaHasAllRequiredElements() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5385406)